src/pages/api/chapters/users/[userId].ts

```diff
@@ -0,0 +1,82 @@
+import type { NextApiResponse } from 'next';
+import { PrismaClient } from '@prisma/client';
+import { ErrorResponse, serverErrorHandler } from '@/utils/error';
+import { NextIronRequest } from '@/utils/session';
+import { getPasswordHash } from '@/utils/password';
+import { validatePassword, validateUsername } from '@/utils/prisma-validation';
+import { withAdminRequestSession } from '@/utils/middlewares/auth';
+
+export type NewCredentialsInput = {
+  newUsername: string;
+  newPassword: string;
+};
+
+async function handler(
+  req: NextIronRequest,
+  res: NextApiResponse<ErrorResponse>,
+) {
+  switch (req.method) {
+    case 'PUT':
+      try {
+        // Check if the provided chapter user exists
+        const prisma = new PrismaClient();
+
+        const { userId } = req.query;
```

Comment:
@Arthelon Since this endpoint is under `chapters/users` does the user id represent `ChapterUser.id` or `User.id` field?